xmtp_mls/src/client.rs

```diff
@@ -560,25 +560,7 @@ where
     }
 
     /// Create a new Direct Message with the default settings
-    pub async fn create_dm(&self, account_address: String) -> Result<MlsGroup<Self>, ClientError> {
-        tracing::info!("creating dm with address: {}", account_address);
-        let provider = self.mls_provider()?;
-
-        let inbox_id = match self
-            .find_inbox_id_from_address(provider.conn_ref(), account_address.clone())
-            .await?
-        {
-            Some(id) => id,
-            None => {
-                return Err(NotFound::InboxIdForAddress(account_address).into());
-            }
-        };
-
-        self.create_dm_by_inbox_id(inbox_id).await
-    }
-
-    /// Create a new Direct Message with the default settings
-    pub async fn create_dm_by_inbox_id(
+    async fn create_dm_by_inbox_id(
```

Comment:
marking this private, since we never really want to be creating a dm when another exists already